src/hotspot/share/classfile/classFileParser.cpp

```diff
@@ -1575,6 +1576,22 @@ void ClassFileParser::parse_fields(const ClassFileStream* const cfs,
     _temp_field_info->adr_at(idx2)->set_index(idx2);
     _static_oop_count++;
   }
+  if (!access_flags().is_identity_class() && !access_flags().is_interface()
+      && _class_name != vmSymbols::java_lang_Object()) {
+    // Acmp map required for abstract and concrete value classes
```

Comment:
Thank you for this comment.  Explains why abstract classes are omitted from above.